openghg/transform/flux/_edgar.py

```diff
@@ -301,21 +331,20 @@ def parse_edgar(
 
     edgar_attrs = edgar_ds.attrs
 
-    # After the data has been extracted and used from the unzipped netcdf
-    # file clean up and remove temporary directory and file.
-    if zipped:
-        temp_extract_folder.cleanup()
-
     # Check for "time" dimension and add if missing.
     flux_ndim = flux_values.ndim
     time_name = "time"
     if time_name in flux_da:
         time = flux_da[time_name].values
+        flux = flux_values  # TODO: this was missing... is this correct?? otherwise 'flux' might be undefined
```

Comment:
@qq23840 added some code to my previous draft EDGAR parser PR to fill NaNs in the flux data with zeroes. That code used a different library to regrid the data, so I'm not sure if it is necessary here. But we should check if regridding to the East Asia domain creates NaNs.